packages/mask/src/social-network/types.ts

```diff
@@ -192,9 +192,11 @@ export namespace SocialNetworkUI {
         export interface NativeCompositionAttachImageOptions {
             recover?: boolean
             relatedTextPayload?: string
+            reply?: boolean
```

Comment:
Replying is only needed by Twitter. Please use a proper name if you need to change the option for all SNS.